src/relay/backend/aot_executor_codegen.cc

```diff
@@ -468,7 +468,14 @@ class AOTExecutorCodegen : public ExprVisitor {
     Target target;
 
     // Handle external function
-    if (func->GetAttr<String>(attr::kCompiler).defined()) {
+    // TODO(@Mousius): Move this to relying only on Target attributes rather than kCompiler
+    auto external_compiler = func->GetAttr<String>(attr::kCompiler).defined();
```

Comment:
either bool or 'is_external_compiler'

Reply:
Now actually contains compiler as a string in latest patch